Adding support for decimeters

arutrub opened the pull request:
Please let me know if there are additional items I need to complete before this is approved. 
All tests pass

Eomm commented on test/best.js:
```diff
@@ -20,10 +20,10 @@ test('excludes measurements', (t) => {
-    unit: 'cm',
-    singular: 'Centimeter',
-    plural: 'Centimeters'
+    val: 12000,
```
Could you update the `exclude` parameter only?

Eomm commented on test/best.js:
```diff
@@ -59,10 +59,10 @@ test('pre-cut off number', (t) => {
-    unit: 'cm',
-    singular: 'Centimeter',
-    plural: 'Centimeters'
+    val: 90,
```
Same here: could you update just the `cutOffNumber` to `100` to get the same expected response?

Eomm reviewed (commented):
I would just leave some changed tests with the old `expected` value, then this PR would be perfect!

Thanks!

arutrub commented:
Thank you for the suggestions! 
I have implemented the requested changes, and I have updated the README to reflect them as well. I left in one for converting to decimeters, if you want me to revert that change, I certainly can! 🙂

Eomm reviewed (approved):
LGTM
